### ISSUE TITLE
Pressing  shift-enter not working when creating chat posts

### ISSUE THREAD

**Issue by tibetsprague** [Hylozoic/hylo-evo#1635]

**Comment by KevinTriplett** [comment]
Can you provide more information to replicate?

On my local dev machine, I created two separate accounts and from three different browsers (FF, Safari, and Chrome) I'm able to shift-enter to create a new line in a chat in all three.

However, I see that Chrome does not auto-update as messages are sent from FF and Safari. FF and Safari seem to update the chat as I create new posts.

**Comment by tibetsprague** [comment]
you're right, its working for me now. maybe what i meant was when editing chat posts. its doing weird things there

**Comment by KevinTriplett** [comment]
Ah okay, I'll take a look at editing and see what's different
